patches/gtsam.patch

```diff
@@ -1,13 +1,92 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8c4229e..5ab20aa 100644
 --- ./CMakeLists.txt
 +++ ./CMakeLists.txt
-@@ -207,7 +207,7 @@ endif()
- ### use our patched version of Eigen
- ### See:  http://eigen.tuxfamily.org/bz/show_bug.cgi?id=704 (Householder QR MKL selection)
- ###       http://eigen.tuxfamily.org/bz/show_bug.cgi?id=705 (Fix MKL LLT return code)
--option(GTSAM_USE_SYSTEM_EIGEN "Find and use system-installed Eigen. If 'off', use the one bundled with GTSAM" OFF)
-+option(GTSAM_USE_SYSTEM_EIGEN "Find and use system-installed Eigen. If 'off', use the one bundled with GTSAM" ON)
```

Comment:
Yes, there is a '-' in the patch, so you are modifying the line (switch default from OFF to ON)